### DISCUSSION TITLE
New Default Port Number for branding

### DISCUSSION THREAD

**Discussion by cmgriffing** [withastro/roadmap#451] [Proposal]
### Body

# Summary

I think the default Port number for Astro in dev should be 4321 to go with the space flight branding.

# Background & Motivation

The existing branding is very space and rocket themed. 4321 would fit this nicely.

It would also make sure that the Astro favicon always shows. When doing dev with other frameworks that use 3000, browsers will often show whatever was cached first. With 4321, Astro would be the only known framework that uses it, so it would usually be the correct favicon.

# Goals

The goal is to add a bit more to the branding of the initial dev experience.

**Comment by natemoo-re** [comment]
I genuinely love this idea—curious what the rest of the team thinks! 🚀

**Reply by FredKSchott** [reply]
oh no, on a personal level I do love this. Very subtle but also super creative!

Definitely a tradeoff between that browser caching issue (which I ran into the other day!) and adding thrash to whatever major this breaking change goes out in. 

Other reason that others have changed: Astro and another project that you are using could both default to 3000, making a conflict much more likely than if we chose our own. Vite changed for this reason.

**Reply by FredKSchott** [reply]
Just an update that a lot of my concerns for thrash were from the v1 -> v2 days, and not wanting to break so soon after our v1 release. Now that time has passed, this concern has gone down somewhat. 

Breaking changes should always be avoided if possible, but shouldn't be avoided at all costs either!

**Comment by FredKSchott** [comment]
https://www.speedguide.net/port.php?port=4321 seems like it's safe, unless you're playing Command & Conquer: Red Alert 3 

![img](https://cdn.cloudflare.steamstatic.com/steam/apps/17480/0000006507.1920x1080.jpg?t=1615390553)

**Reply by Princesseuh** [reply]
(which you shouldn't, since Red Alert 2 is just the best C&C)

**Comment by matthewp** [comment]
I do kind of like this idea.

**Comment by ElianCodes** [comment]
Never thought about this! This is great! 🚀🚀🚀

**Comment by ematipico** [comment]
I do like this idea too!

**Comment by delucis** [comment]
Astro’s default port was changed to `4321` as part of the Astro v3 release! 🚀
